cfgov/unprocessed/css/organisms/video-player.less

```diff
@@ -366,6 +365,7 @@ button.o-video-player_play-btn,
             height: 100%;
             width: 100%;
             opacity: 1;
+            border: 0;
```

Comment:
This moves `border: 0` from inside `o-video-player_video-container__flexible` (standalone player) to inside `o-featured-content-module` (featured content video). So it ends up with a border on the standalone player:
<img width="959" alt="Screen Shot 2021-09-29 at 9 44 32 AM" src="https://user-images.githubusercontent.com/704760/135281210-8ce7a7eb-f8ec-4f7d-a54d-aa9a9b6aae4f.png">

It needs to work on both /admin/template_debug/v1/video_player/ and /admin/template_debug/v1/featured_content/